docs/examples/data-access.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accessing Data\n",
+    "\n",
+    "There are many different conventions used to define unstructured grids (UGRID, SCRIP, etc.). These conventions often have different variable names used to describe the same variables.\n",
+    "\n",
+    "UXarray offers multiple ways of accessing data stored in `Grid` objects. This notebook will highlight different ways of accessing data with examples both in UGRID and non-UGRID formats.\n",
```

Comment:
Tell something about the facts that 

(1) whatever the grid type UXarray reads in, it converts it to a UGRID type in the Grid object.  
(2) The UGRID convention is flexible about variable namings except a few required topology attributes (such as "cf_role", "topology_dimension", etc.)
(3) To handle all of those variable names coming from either non-UGRID or UGRID types, UXarray Grid object makes use of a dictionary in which it uses a set of standardized names for keys and custom names as values.
      - There are a few different ways to access those variables in UXarray Grid object.